lisa/tools/ls.py

```diff
@@ -99,7 +99,7 @@ def path_exists(self, path: str, sudo: bool = False) -> bool:
         return output.strip() == "True"
 
     def list(self, path: str, sudo: bool = False) -> List[str]:
-        command = f'Get-ChildItem -Path "{path}" `| Select-Object -ExpandProperty Name'
+        command = f'Get-ChildItem -Path "{path}" | Select-Object -ExpandProperty Name'
```

Comment:
There were some changes forth and back on how to encoding the string. Please make sure it's right way and no regression on other places.

Reply:
Do you recall the reason behind the back tick (`)? Because during my testing it just throws an error. "Get-ChildItem: A positional parameter cannot be found that accepts argument 'Select-Object'"

My understanding is that backtick is used for escaping things. Here, we don't want to escape `|`. It needs to be interpreted as the pipe operator instead of a literal `|`.

Reply:
It's used to escape `|`, because there was several layers of text wrappers. But after I encode the command to base64, it may not be a concern.

Reply:
https://github.com/microsoft/lisa/pull/3159